src/UserSpaceInstrumentation/InjectLibraryInTracee.cpp

```diff
@@ -0,0 +1,298 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "InjectLibraryInTracee.h"
+
+#include <absl/base/casts.h>
+#include <absl/strings/match.h>
+#include <absl/strings/str_format.h>
+#include <dlfcn.h>
```

Comment:
is this needed?

Reply:
No, left over from previous version ... removed.